proto/proto/eraftpb.proto

```diff
@@ -72,12 +72,18 @@ message Message {
     MessageType msg_type = 1;
     uint64 to = 2;
     uint64 from = 3;
+    // sender's term
     uint64 term = 4;
-    uint64 log_term = 5;
-    uint64 index = 6;
+    // index of log entry immediately preceding new ones
```

Comment:
not quite understand the meaning

Reply:
> index of log entry immediately preceding new ones

this is actually the original description in Raft paper Figure 2, refer to `AppendEntries RPC -> prevLogIndex`.

However, in my understanding, this `index` field also stands for `lastLogIndex` in `RequestVote RPC`.

Reply:
to be honest, one message format to combine different message type confused me a lot, since the field name is not accurate. I got it clear until I read the paper.